src/test/java/org/broadinstitute/hellbender/tools/walkers/varianteval/evaluators/TestEvaluatorWhichRequiresReferenceButDoesntSayItDoes.java

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.hellbender.tools.walkers.varianteval.evaluators;
```

Comment:
I'm concerned that this test class will be picked up by the reflection code in the main tool and exposed to the user, since it's in the standard evaluator package.

Reply:
That's a reasonable concern, but this is in the test code, so it should not be available to the main tool at runtime.